fmriprep/cli/run.py

```diff
@@ -35,18 +35,11 @@ def main():
 
     from ..utils.bids import write_bidsignore, write_derivative_description
     from .parser import parse_args
-    from .workflow import build_workflow
 
     parse_args()
 
-    if "pdb" in config.execution.debug:
-        from fmriprep.utils.debug import setup_exceptionhook
-
-        setup_exceptionhook()
-        config.nipype.plugin = "Linear"
-
     sentry_sdk = None
-    if not config.execution.notrack and not config.execution.debug:
+    if not config.execution.notrack:
```

Comment:
@oesteban I think the issue is that not all sentry calls are hidden with `config.execution.deubg`.